server/fleet/activities.go

```diff
@@ -895,10 +895,12 @@ func (a ActivityTypeFleetEnrolled) Documentation() (activity string, details str
 }
 
 type ActivityTypeMDMEnrolled struct {
-	HostSerial       string `json:"host_serial"`
-	HostDisplayName  string `json:"host_display_name"`
-	InstalledFromDEP bool   `json:"installed_from_dep"`
-	MDMPlatform      string `json:"mdm_platform"`
+	HostSerial       *string `json:"host_serial"`
+	EnrollmentID     *string `json:"enrollment_id"`
```

Comment:
Just a clarifying comment for myself, we switched to pointers here now, because one of these will be nil, where if we did not use pointers it would show up as empty strings and be confusing?